seatunnel-common/src/main/java/org/apache/seatunnel/common/config/CheckConfigUtil.java

```diff
@@ -72,6 +72,16 @@ public static CheckResult checkAtLeastOneExists(Config config, String... params)
         }
     }
 
+    public static boolean isInvalidParam(Config config, String param) {
+        boolean isInvalidParam = false;
+        if (!config.hasPath(param) || config.getAnyRef(param) == null) {
```

Comment:
can be simplified, `config.hasPath()` can ensure `config.getAnyRef(param) != null`

Reply:
what about if key exists, but value is null

Reply:
as javadoc https://lightbend.github.io/config/latest/api/index.html says, it will return false:

```
Checks whether a value is present and non-null at the given path. This differs in two ways from Map.containsKey() as implemented by [ConfigObject](https://lightbend.github.io/config/latest/api/com/typesafe/config/ConfigObject.html): it looks for a path expression, not a key; and it returns false for null values, while containsKey() returns true indicating that the object contains a null value for the key.
```

Reply:
make sense, thx.